--- docker-compose-1.7.1/setup.py~	2016-05-04 21:52:57.000000000 +0300
+++ docker-compose-1.7.1/setup.py	2016-05-19 14:24:16.408589040 +0300
@@ -31,7 +31,7 @@
     'cached-property >= 1.2.0, < 2',
     'docopt >= 0.6.1, < 0.7',
     'PyYAML >= 3.10, < 4',
-    'requests >= 2.6.1, < 2.8',
+    'requests >= 2.6.1',
     'texttable >= 0.8.1, < 0.9',
     'websocket-client >= 0.32.0, < 1.0',
     'docker-py >= 1.8.1, < 2',
@@ -47,8 +47,8 @@
 
 
 if sys.version_info[:2] < (3, 4):
-    tests_require.append('mock >= 1.0.1')
-    install_requires.append('enum34 >= 1.0.4, < 2')
+    tests_require.append('mock')
+    install_requires.append('enum34')
 
 
 setup(
