--- docker-compose-1.27.0/requirements.txt.orig	2020-09-07 22:15:24.818988058 +0200
+++ docker-compose-1.27.0/requirements.txt	2020-09-07 22:17:28.372476931 +0200
@@ -1,23 +1,22 @@
-backports.shutil_get_terminal_size==1.0.0
-cached-property==1.5.1
-certifi==2020.6.20
-chardet==3.0.4
-colorama==0.4.3; sys_platform == 'win32'
-distro==1.5.0
-docker==4.3.1
-docker-pycreds==0.4.0
-dockerpty==0.4.1
-docopt==0.6.2
-idna==2.10
-ipaddress==1.0.23
-jsonschema==3.2.0
-paramiko==2.7.1
-pypiwin32==219; sys_platform == 'win32' and python_version < '3.6'
-pypiwin32==223; sys_platform == 'win32' and python_version >= '3.6'
-PySocks==1.7.1
-python-dotenv==0.14.0
-PyYAML==5.3.1
-requests==2.24.0
-texttable==1.6.2
-urllib3==1.25.10; python_version == '3.3'
-websocket-client==0.57.0
+backports.shutil_get_terminal_size
+cached-property
+certifi
+chardet
+colorama; sys_platform == 'win32'
+distro
+docker
+docker-pycreds
+dockerpty
+docopt
+idna
+ipaddress
+jsonschema
+paramiko
+pypiwin3; sys_platform == 'win32'
+PySocks
+python-dotenv
+PyYAML
+requests
+texttable
+urllib3; python_version == '3.3'
+websocket-client
--- docker-compose-1.27.0/setup.py.orig	2020-09-07 22:15:24.818988058 +0200
+++ docker-compose-1.27.0/setup.py	2020-09-07 22:19:17.642592487 +0200
@@ -25,17 +25,17 @@
 
 
 install_requires = [
-    'cached-property >= 1.2.0, < 2',
-    'docopt >= 0.6.1, < 1',
-    'PyYAML >= 3.10, < 6',
-    'requests >= 2.20.0, < 3',
-    'texttable >= 0.9.0, < 2',
-    'websocket-client >= 0.32.0, < 1',
-    'distro >= 1.5.0, < 2',
-    'docker[ssh] >= 4.3.1, < 5',
-    'dockerpty >= 0.4.1, < 1',
-    'jsonschema >= 2.5.1, < 4',
-    'python-dotenv >= 0.13.0, < 1',
+    'cached-property >= 1.2.0',
+    'docopt >= 0.6.1',
+    'PyYAML >= 3.10',
+    'requests >= 2.20.0',
+    'texttable >= 0.9.0',
+    'websocket-client >= 0.32.0',
+    'distro >= 1.5.0',
+    'docker[ssh] >= 4.3.1',
+    'dockerpty >= 0.4.1',
+    'jsonschema >= 2.5.1',
+    'python-dotenv >= 0.13.0',
 ]
 
 
@@ -49,9 +49,9 @@
     tests_require.append('mock >= 1.0.1, < 4')
 
 extras_require = {
-    ':python_version < "3.5"': ['backports.ssl_match_hostname >= 3.5, < 4'],
-    ':sys_platform == "win32"': ['colorama >= 0.4, < 1'],
-    'socks': ['PySocks >= 1.5.6, != 1.5.7, < 2'],
+    ':python_version < "3.5"': ['backports.ssl_match_hostname >= 3.5'],
+    ':sys_platform == "win32"': ['colorama >= 0.4'],
+    'socks': ['PySocks >= 1.5.6, != 1.5.7'],
     'tests': tests_require,
 }
 
